### DISCUSSION TITLE
Does it support backing up data to PVC in the same cluster?

### DISCUSSION THREAD

**Discussion by acechef** [k8up-io/k8up#771] [Q&A] [answered]
Does it support backing up data to PVC in the same cluster?

**Comment by Kidswiss** [comment, marked as answer]
Hi @acechef 

You could deploy minio (https://min.io/) on your cluster and use that as a S3 target for K8up.

I personally would not do that. Because if you have issues with your k8s storage and loose all data, then your backup will be gone, too.

**Reply by acechef** [reply]
Thanks Reply
